modules/compliance-operator-configure-scanning.adoc

```diff
@@ -10,7 +10,9 @@ Create a `ScanSettingBinding` object in the `openshift-compliance` namespace to
 
 [NOTE]
 ====
-This example uses `cis` and `cis-node` profiles, but {ocp} provides additional profiles. See "Understanding the Compliance Operator" in the "Additional resources" section for more information.
+* If you are using Compliance (2.0), you can schedule your scan manually instead of creating the `ScanSettingBinding` on the Compliance Operator.
+
+* This example uses `cis` and `cis-node` profiles, but {ocp} provides additional profiles. See "Understanding the Compliance Operator" in the "Additional resources" section for more information.
```

Comment:
The `cis` and `cis-node` profiles don't exist, but instead use `ocp4-cis` and `ocp4-cis-node`.